exp-player/addon/helpers/breaklines.js

```diff
@@ -0,0 +1,10 @@
+// h/t http://stackoverflow.com/questions/12331077/does-handlebars-js-replace-newline-characters-with-br
```

Comment:
Drawing from the Stackoverflow answer: 

What is the advantage of adding this helper rather than using a single [CSS style](https://developer.mozilla.org/en-US/docs/Web/CSS/white-space)?